### DISCUSSION TITLE
Why am I getting extra line fields while writing to output file in EDI/X12 format?

### DISCUSSION THREAD

**Discussion by sharmadeepak69** [nextgenhealthcare/connect#5086] [Q&A]
I am using database reader as source and batch splitting based on each record and then transforming that to 834 format and writing on the output file, so when I am writing onto the output file after processing each batch, mirth is appending couple of line fields on that where as per my use case i want continuous data.

For example,

What i am getting in the output file:
1111111

2222222

3333333

vs What I want in the output file:
1111111
2222222
3333333

**Comment by pacmano1** [comment]
Always hard to help when you post **no** code that shows the message being built.
